packages/riverpod_cli/README.md

```diff
@@ -21,7 +21,7 @@ Add to your `pubspec.yaml`:
 
 ```yaml
 dev_dependencies:
-  codemod_riverpod: ^0.0.2
```

Comment:
Wait this was supposed to say "pub global activate"
I think I lost it when fixing conflicts 🤔

Reply:
I think you can do either. If not we will probably end up getting some issue about it.